packages/ssx-sdk/src/modules/Encryption.ts

```diff
@@ -1,5 +1,6 @@
 import * as jose from 'jose';
-import { IUserAuthorization } from './UserAuthorization';
+import { IUserAuthorization, UserAuthorization } from './UserAuthorization';
+import * as LitJsSdk from '@lit-protocol/lit-node-client';
```

Comment:
I'm receiving this warning after the compilation:

```
WARNING in ../../node_modules/@lit-protocol/ecdsa-sdk/src/lib/ecdsa-sdk.js 428:22-61
Critical dependency: the request of a dependency is an expression
 @ ../../node_modules/@lit-protocol/ecdsa-sdk/src/index.js 4:21-47
 @ ../../node_modules/@lit-protocol/crypto/src/lib/crypto.js 7:18-52
 @ ../../node_modules/@lit-protocol/crypto/src/index.js 4:21-44
 @ ../../node_modules/@lit-protocol/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.js 9:17-48
 @ ../../node_modules/@lit-protocol/lit-node-client-nodejs/src/index.js 6:29-68 18:21-60
 @ ../../node_modules/@lit-protocol/lit-node-client/src/index.js 25:21-68
 @ ./src/modules/Encryption.ts 37:30-70
 @ ./src/modules/index.ts 20:13-36
 @ ./src/index.ts 19:13-33
```

And this is generating a warning on ssx-test-dapp:
```
Compiled with warnings.

Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.

WARNING in ../../packages/ssx-sdk/dist/index.js 59504:80-87
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

webpack compiled with 1 warning
```

@skgbafa or @w4ll3 have you ever seen this?